src/agent/application.cpp

```diff
@@ -43,8 +43,8 @@
 
 namespace otbr {
 
-bool                 Application::sShouldTerminate = false;
-const struct timeval Application::kPollTimeout     = {10, 0};
+std::atomic_bool     Application::sShouldTerminate{false};
```

Comment:
I think the old "= false" initialization is better.

Reply:
It does not compile.

Reply:
OK, it doesn't compile without mandatory copy elision, which was introduced in C++17.

Then I think it's better to use the traditional constructor syntax (with parentheses) in this case.

Reply:
> OK, it doesn't compile without mandatory copy elision, which was introduced in C++17.

Thanks for the insight. 


> Then I think it's better to use the traditional constructor syntax (with parentheses) in this case.

Is there a explanation why which is better?